lodash/lodash.d.ts

```diff
@@ -6412,6 +6412,7 @@ declare module _ {
         endsWith(str?: string, target?: string, position?: number): boolean;
         escape(str?: string): string;
         escapeRegExp(str?: string): string;
+		isMatch(str?: string, target?: string): boolean;
```

Comment:
Yes, I left out the `customizer` and the `thisArg` because this is only being used for string comparisons.